window.py

```diff
@@ -9,62 +9,66 @@
 
 LOGFILE = "sezam.log"
 
-# TODO: supply intelligent defaults
-local_samase = ""
-unpack_samase = ""
-
 def pathSet(value: str) -> bool:
     if value == "":
         print("Attempted to perform an operation without a directory selected!")
         return False
     return True
-    
-# Log setup
-logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', filename=LOGFILE, encoding='utf8', level = logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
-logging.FileHandler(LOGFILE, mode = "w")
 
-# Window layout
-layout = [[gui.Button("Select build directory"), gui.Text("Please select a path", key="curPath"), gui.Push(), gui.Checkbox("Run in x64", key="x64")],
-          [gui.Push(), gui.Button("Launch"), gui.Button("Build From"), gui.Push()],
-          [gui.Push()],
-          [gui.Button("Select .exe to Unpack"), gui.Text("No exe selected", key="curExe")],
-          [gui.Push(), gui.Button("Unpack from"), gui.Push()]
-]
+class Window():
+    def __init__(self):
+        # TODO: supply intelligent defaults
+        self.local_samase = ""
+        self.unpack_samase = ""
+
+        layout = [[gui.Button("Select build directory"), gui.Text("Please select a path", key="curPath"), gui.Push(), gui.Checkbox("Run in x64", key="x64")],
```

Comment:
We don't have to have dangling layout state, it cleans up after `__init__`, unlike before. The important vars are on the `self` (instance of the object). Having less variables in scope reduces mental overhead of reading the code.